week-1-practise/level-2/3-currency-conversion.js

```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(price) {
```

Comment:
One small thing to think about: having good function parameter names is important. When other developers on your team see your functions, it can help them to understand how to use them. The names should make it clear what input is expected.

In this case, the input must be in GBP (pounds), otherwise the conversion does not work. How could you change the function parameter name (which you called price) to make this clearer?